### DISCUSSION TITLE
tv-series/the-traitors-canada-season-2-episode-3/

### DISCUSSION THREAD

**Discussion by giscus** [telepisodes/projectfreetvlol-comments#7] [General]
# tv-series/the-traitors-canada-season-2-episode-3/

Watch series in ProjectFreeTv  and view all of movies online at ProjectFreeTv, your top site for streaming entertainment online.

https://www.projectfreetv.lol/tv-series/the-traitors-canada-season-2-episode-3/

**Comment by Rushends** [comment]
The audio and video is misaligned in this episode. Can't follow a thing
